tests/KMS.test.ts

```diff
@@ -1133,3 +1149,56 @@ test(
     timeout: 30 * 1000,
   },
 )
+
+test(
+  "Grant and revoke Access",
+  async () => {
+    const kmsToken2 =
+      "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IjVVU1FrSVlULW9QMWZrcjQtNnRrciJ9.eyJuaWNrbmFtZSI6ImNsaWVudDIiLCJuYW1lIjoiY2xpZW50MkBjb3NtaWFuLmNvbSIsInBpY3R1cmUiOiJodHRwczovL3MuZ3JhdmF0YXIuY29tL2F2YXRhci83MjZhNDZkMDdkZjY0Y2VjMDc4ZTQzNzUyYjg5MjFmMz9zPTQ4MCZyPXBnJmQ9aHR0cHMlM0ElMkYlMkZjZG4uYXV0aDAuY29tJTJGYXZhdGFycyUyRmNsLnBuZyIsInVwZGF0ZWRfYXQiOiIyMDIzLTA5LTI2VDEzOjAwOjE3LjIyNVoiLCJlbWFpbCI6ImNsaWVudDJAY29zbWlhbi5jb20iLCJlbWFpbF92ZXJpZmllZCI6ZmFsc2UsImlzcyI6Imh0dHBzOi8va21zLWNvc21pYW4uZXUuYXV0aDAuY29tLyIsImF1ZCI6IkszaXhldXhuVDVrM0Roa0tocWhiMXpYbjlFNjJGRXdJIiwiaWF0IjoxNjk1NzMzMjE5LCJleHAiOjE2OTYwOTMyMTksInN1YiI6ImF1dGgwfDY1MDQ2YzdlNWEwYWZmYzg3MjAxZWE1ZiIsInNpZCI6IkstTkdIVDFqRFJRMzFiY0ZqRHdpYnZlMkRHbGxlZzZqIiwibm9uY2UiOiJaMmR5V25GMVlucG1iSE5IUmtRNGNVbzRTRXRZUVdOdFlqZFNZelZmWVU1b1l6QmxjMFUwY3pCaE53PT0ifQ.QEVI0R3LvIJuI1m3Ew0LWk29uuSF5ObznWIWw9ctlnj8BqzwUmfCs7axDpTtRkMmKkTMq3Z0EzhpefTC_pIBntA739fJMfKdO2pGBVbBVkR5r86XyELbFeqJZNEFvq3WOmSgUtjLtE121NDkaMgZr2LxEiG6d3daD2wYQ9Ust_-JxvqDRD1Ted5-jc4w4YYir9k9lnrUZIxYrUiEKKnNWkwWZP2JHnrzB_zsgwcE4K1Hii2Q2QBT_LiOawgOnf_tbX59M-3qvQXyEfAJ-iWrHKIvDP3qkf4BUVqGkiotbsCMMntyCfgwm35Wzmo2nunw72i-4bkQjh9Cz9GpikFWDA"
+
+    const client = new KmsClient(
+      `http://${process.env.KMS_HOST || "localhost"}:9998`,
+      kmsToken,
+    )
+    if (!(await client.up())) {
+      console.error("No KMIP server. Skipping test")
+    }
```

Comment:
```suggestion
```

IMHO we should fail all the KMS tests where the KMS server is required and not up.

Reply:
Yes - I put the check in a beforeAll function to setup the client. So the check is only done once